ui_components/src/travel/Map.js

```diff
@@ -1,5 +1,5 @@
 
-export const Map = ({mapData}) => {
+export const Map = ({ mapData, scoutData, playerId }) => {
```

Comment:
Could we add Flow types here to define the shape of these props? Can be a follow-up, I can help if needed

Reply:
Appreciated!